src/types/system.ts

```diff
@@ -1,5 +1,7 @@
 import { AUTH_MODE, CACHE_TTL, NODE_TYPES, SCOPE_GROUP } from 'common/enums'
 
+const UPLOAD_RATE_LIMIT = 40 // 20 pictures per 12 minutes, `directImageUpload` called twice for each picture
```

Comment:
20？

Reply:
as stated in comment, `directImageUpload` is called twice for each picture